src/rdkafka_request.c

```diff
@@ -3119,6 +3128,7 @@ rd_kafka_AlterConfigsRequest (rd_kafka_broker_t *rkb,
                             "AlterConfigs.incremental=true (KIP-248) "
                             "not supported by broker, "
                             "requires broker version >= 2.0.0");
+                rd_kafka_replyq_destroy(&replyq);
```

Comment:
There is another error return below on line 3170, can you add this destroy there too?

Reply:
missed that, have added it.